sdk/ml/azure-ai-ml/azure/ai/ml/_schema/registry/registry.py

```diff
@@ -12,7 +12,7 @@
 from azure.ai.ml.entities._registry.registry_support_classes import SystemCreatedAcrAccount
 from azure.ai.ml._utils._experimental import experimental
 
-from azure.ai.ml._schema.workspace.identity import IdentitySchema
```

Comment:
Please continue to use workspace.identity schema. We do not want to change the CLI GAed experience.

Reply:
It's all been removed, we are currently using the compute.identity, it's trying to follow your change...
In the last PR, you introduced the identity->List of UAI, instead of a dictionary.
So all GAed experience about the identity would be same as compute.identity.

Reply:
Sorry for the confusion. I meant follow what compute is doing but for CLI we will need to continue using workspace identity schema since that contract has been GAed there. We cannot change it now.